tslib/readers/pi_xml_reader.py

```diff
@@ -146,8 +146,166 @@ def get_series(self):
 
             yield metadata, dataframe
 
+    def bulk_get_series(self, chunk_size=250000):
```

Comment:
Ik begrijp dat het verschil tussen `get_series` en `bulk_get_series` hem in de chunking zit? Of zijn er nog andere verschillen?

Reply:
Voor het archiedf: Bulk (Meta)Dataframe bevat ook kolommen waardoor de timeseries van elkaar te onderscheiden zijn. En de metadata is een DataFrame.